clients/feeder/feeder.go

```diff
@@ -214,7 +214,7 @@ func (c *Client) get(ctx context.Context, queryURL string) (io.ReadCloser, error
 			if wait > c.maxWait {
 				wait = c.maxWait
 			}
-			c.log.Warnw("failed query to feeder, retrying...", "retryAfter", wait.String())
+			c.log.Warnw("failed query to feeder", "getting error", err, "retrying...", "retryAfter", wait.String())
```

Comment:
```suggestion
			c.log.Warnw("Failed to query feeder, retrying...", "retryAfter", wait.String(), "err", err)
```
I think we need to set a timeout on the request's context. Otherwise it will just wait on the server forever.